py-polars/polars/meta/versions.py

```diff
@@ -68,12 +68,14 @@ def _get_dependency_list() -> list[str]:
     return [
         "adbc_driver_manager",
         "altair",
+        "boto3",
         "cloudpickle",
         "connectorx",
         "deltalake",
         "fastexcel",
         "fsspec",
         "gevent",
+        "google.auth",
```

Comment:
drive-by print the `boto3` and `google-auth` package versions in `show_versions()`, since we could be using them now